content/index.fr.md

```diff
@@ -0,0 +1,230 @@
+---
+# Translation info https://www.w3.org/wiki/WAI/Website/Translate
+
+title: "Vue d'ensemble de l'évaluation de l'accessibilité Web"
+nav_title: "Vue d'ensemble : Evaluation"
+
+lang: fr
+last_updated: 2020-12-27
+permalink: /test-evaluate/fr
+
+translators: 
+- name: "Sofia Ahmed"
+contributors:
+- name: "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-eval-overview
+  path: content/index.fr.md
+
+feedbackmail: wai@w3.org
+class: tight-page
+footer: >
+  <p>Note à propos de l'audiodescription : les vidéos présentes sur cette page n'incluent pas d'audiodescription synchronisée car les images n'illustrent que l'audio et ne fournissent pas d'informations supplémentaires. Dans ce cas-ci, l'audiodescription serait plus gênante qu'utile pour la plupart des utilisateurs, y compris les personnes qui ne peuvent pas voir les images. La description des informations contenues dans les images est reprise dans la transcription avec la description des images ("transcription descritive").</p>
+  <p><strong>Date :</strong> Mise à jour : 28 avril 2020.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Vidéos développées par le<em>Groupe de travail Éducation et Promotion</em> (<a href="http://www.w3.org/WAI/EO/">EOWG</a>) avec le soutien du projet <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide</a> financé par la Commission européenne (CE) dans le cadre du programme Horizon 2020 (822245) <a href="./acknowledgements/">Remerciements</a>.</p>
```

Comment:
Add space separation in French for "leGroupe". It should be "le Groupe"

Reply:
I concur 👍 
- We also do not need `<em></em>` : not in the original version and not needed in French.
- No capital letter is needed at the beginning of "groupe".